hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -310,6 +318,20 @@ private FlinkOptions() {
           + "Actual value will be obtained by invoking .toString() on the field value. Nested fields can be specified using "
           + "the dot notation eg: `a.b.c`");
 
+  public static final ConfigOption<String> INDEX_KEY_FIELD = ConfigOptions
+      .key(HoodieIndexConfig.BUCKET_INDEX_HASH_FIELD.key())
+      .stringType()
+      .defaultValue("uuid")
```

Comment:
hmm not quite sure we should do that here. The user need some basic knowledge about this index type, so they know how to operate the table. e.g. how to handle the bucket number and how does this hash key field work.

Reply:
> If the index key field not set, use the record key field as default to make the configuration more user-friendly?

A good suggestion, i would prefer this way so that user do not need to declare the keys twice. The bucket number is a little different i think and it can have a default value.

Reply:
ok, two vote to define this. fixed